client/dashboard/profile-wizard/steps/start/index.js

```diff
@@ -54,7 +54,7 @@ class Start extends Component {
 	async skipWizard() {
 		const { createNotice, isProfileItemsError, updateProfileItems } = this.props;
 
-		await updateProfileItems( { skipped: true, wcs_jetpack: 'skipped' } );
+		await updateProfileItems( { wcs_jetpack: 'skipped' } );
```

Comment:
Wondering if instead of `wcs_jetpack` we could use something like `plugins_skipped`.  In this case I only skipped WCS and not Jetpack.  This would also leave room to grow if we add additional plugins.

Reply:
This param is also used to detect if the plugins are already installed (to skip the start page), so `plugins_skipped` wouldn't work, but I agree the naming could be better than `wcs_jetpack`.

I renamed this to `plugins` in https://github.com/woocommerce/woocommerce-admin/pull/3040/commits/ef3e3fe5bd7f28295869b1ff60f52c9fd7affef8, and improved our tracking a bit. Do you mind taking another look since I doubled the line count of the PR?